File: WorldFeed/Server/src/WorldFeed.Common/Events/LimitReachedEventArgs.cs
namespace WorldFeed.Common.Events
{
    using System;

    /// <summary>
    /// Event to inform that the limit of tweets that can be received by the application has been reached.
    /// </summary>
    public class LimitReachedEventArgs : EventArgs
    {
        public LimitReachedEventArgs(int numberOfTweetsNotReceived)
        {
            this.NumberOfTweetsNotReceived = numberOfTweetsNotReceived;
        }

        public int NumberOfTweetsNotReceived { get; }
    }
}

File: WorldFeed/Client/src/app/core/Public/Events/LimitReachedEventArgs.ts
import {EventArgs} from "../../../c#-objects/TypeScript.NET-Core/packages/Events/source/EventArgs";

// Event to inform that the limit of tweets that can be received by the application has been reached.
export class LimitReachedEventArgs extends EventArgs {
  constructor(numberOfTweetsNotReceived: number) {
    super();
    this.NumberOfTweetsNotReceived = numberOfTweetsNotReceived;
  }

  public NumberOfTweetsNotReceived: number;
}
